packs/validator/values.yaml

```diff
@@ -0,0 +1,578 @@
+pack:
+  content:
+    images:
+      - image: quay.io/validator-labs/validator:v0.0.40
+      - image: quay.io/validator-labs/validator-plugin-aws:v0.0.26
+      - image: quay.io/validator-labs/validator-plugin-azure:v0.0.11
+      - image: quay.io/validator-labs/validator-plugin-kubescape:v0.0.3
+      - image: quay.io/validator-labs/validator-plugin-network:v0.0.16
+      - image: quay.io/validator-labs/validator-plugin-oci:v0.0.10
+      - image: quay.io/validator-labs/validator-plugin-vsphere:v0.0.20
+      - image: quay.io/validator-labs/validator-certs-init:1.0.0
+      - image: gcr.io/spectro-images-public/release/spectro-cleanup:1.2.0
+    charts:
+      - repo: https://validator-labs.github.io/validator
+        name: validator
+        version: v0.0.40
```

Comment:
TODO: add remaining charts for the plugins

Reply:
And add the following to pack.json:
```yaml
"annotations": {
    "upload_helm_charts": "true",
},
```